companies/4wave.json

```diff
@@ -6,7 +6,7 @@
     "categories": [
         "ads"
     ],
-    "name": "blueleads GmbH",
+    "name": "  blueleads GmbH      ",
```

Comment:
🚫 **[recordlint]** <[trim-string-fields](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Value of string field `name` is not trimmed (it contains leading or trailing whitespace).
```suggestion
    "name": "blueleads GmbH",
```